modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -0,0 +1,242 @@
+        #intel.com/intel_fec_lte: '1'
+      limits:
+        memory: 4Gi
+        cpu: "4"
+        hugepages-1Gi: 4Gi
+        intel.com/intel_fec_5g: '1'
+        #intel.com/intel_fec_acc100: '1'
+        #intel.com/intel_fec_lte: '1
+----
+<1> Specify the `namespace` you created in step 1.
+<2> This defines the test image containing the compiled DPDK.
+<3> Make the container execute internally as the root user.
+<4> Specify hugepage size hugepages-1Gi and the quantity of hugepages that will be allocated to the pod. Hugepages and isolated CPUs needs to be configure using the Performance Addon Operator.
+<5> Specify the number of CPUs.
+<6> Testing of the N3000 5G FEC configuration is supported by `intel.com/intel_fec_5g`.
++
+[NOTE]
+====
+To test the ACC100 configuration, uncomment `intel.com/intel_fec_acc100` by removing the # symbol.
+To test the N3000 4G/LTE configuration, uncomment `intel.com/intel_fec_lte` by removing the # symbol.
+Only one resource should be active at any time.
+====
+
+. Create the pod:
++
+[source,terminal]
+----
+$  oc apply -f pod-test.yaml
+----
+
+. Check that the pod is created:
++
+[source,terminal]
+----
+$  oc get pods -n test-bbdev
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY           STATUS          RESTARTS        AGE
+pod-bbdev-sample-app                            1/1             Running         0               80s
+----
+
+. Remote shell log in to the `pod-bbdev-sample-app`:
++
+[source,terminal]
+----
+$ oc rsh pod-bbdev-sample-app
+----
++
+.Example output
+[source,terminal]
+----
+sh-4.4#
+----
+
+. Print a list of environment variables:
++
+[source,terminal]
+----
+sh-4.4# env
+----
++
+.Example output
+[source,terminal]
+----
+
+N3000_CONTROLLER_MANAGER_METRICS_SERVICE_PORT_8443_TCP_ADDR=172.30.133.131
+SRIOV_FEC_CONTROLLER_MANAGER_METRICS_SERVICE_PORT_8443_TCP_PROTO=tcp
+DPDK_VERSION=20.11
+PCIDEVICE_INTEL_COM_INTEL_FEC_5G=0.0.0.0:1d.00.0 <1>
+~/usr/bin/env
+HOSTNAME=fec-pod
+
+----
++
+<1> This is the PCI address of the virtual function.
++
+[NOTE]
+====
+Depending on the resource that the user requests in the `pod-test.yaml` file, this can be any one of following three PCI addresses:
+
+* PCIDEVICE_INTEL_COM_INTEL_FEC_ACC100
+* PCIDEVICE_INTEL_COM_INTEL_FEC_5G
+* PCIDEVICE_INTEL_COM_INTEL_FEC_LTE
+====
+
+
+.  Change to the `test-bbdev` directory.
++
+[source,terminal]
+----
+sh-4.4# cd test/test-bbdev/
+----
+
+. Check the CPUs that are assigned to the pod:
++
+[source,terminal]
+----
+sh-4.4# export CPU=$(cat /sys/fs/cgroup/cpuset/cpuset.cpus)
+sh-4.4# echo ${CPU}
+----
+This prints out the CPUs that are assigned to the `fec.pod`.
++
+.Example output
+[source,terminal]
+----
+24,25,64,65
+----
+
+. Run the `test-bbdev` application to test the device:
++
+[source,terminal]
+----
+sh-4.4# ./test-bbdev.py -e="-l ${CPU} -a ${PCIDEVICE_INTEL_COM_INTEL_FEC_5G}" -c validation \ -n 64 -b 32 -l 1 -v ./test_vectors/*"
```

Comment:
Please make sure that you meant to include the backslash in the command.  Maybe the original text used that to add a newline and show the command on two lines.